mdm/api/src/main/java/com/gentics/mesh/core/data/search/IndexHandler.java

```diff
@@ -49,6 +52,14 @@ public interface IndexHandler<T extends HibBaseElement> {
 	 */
 	Function<String, T> elementLoader();
 
+
+	/**
+	 * Returns loader function which can be used to find and load multiple elements by uuids.
```

Comment:
Please add to the comment, how implementations should behave with respect to the following topics:
* Order of the returned elements (will it reflect the order of the given uuids?) If the order is relevant, shouldn't we pass a List<String>?
* Handling of duplicates (if the given collection contains the same UUIDs multiple times, will the stream contain the found object multiple times)
* Permission checks (will permissions be checked?)
* Handling of unknown UUIDs 

It would also be good to have Unit Tests for all implementations to show the expected behaviour

Reply:
The permission behavior is kept from the single-item fetcher - i.e. no visible perm checks were, and are applied.

Reply:
The order preservation is applied at the client side.